### DISCUSSION TITLE
Trouble publishing repository to Pages

### DISCUSSION THREAD

**Discussion by DebbieU** [community/community#30728] [Other Feature Feedback, Questions, & Ideas] [answered]
I have several repositories on GitHub. I want to publish them each separately as websites. I made the repository public. I go to the Pages tab and choose to Deploy from a branch, using the Main  /root branch and save that. I don't see any place that says "Your site is live". However if I go to Custom Domain, and type in the repository name and save that, it gives me an error "The custom domain `star` is not properly formatted. See https://docs.github.com/articles/troubleshooting-custom-domains/#github-repository-setup-errors for more information."  This afternoon when I refreshed the page, even though it had the error, it then said "Your site is live at https://yourname.github.io/test-repo/”.  However, this evening when I do the same thing with another repository, it gives me the same error, but when I refresh it doesn't say "your site is live".  What am I missing? This repository is also Public.

**Comment by radar-blip** [comment, marked as answer]
The one site is working. 
The other is not. I'm not sure why yet.

**Reply by radar-blip** [reply]
I see you have your index.html in place (for the other site) but I cant figure out why its not deploying. 
Maybe someone else will chime in and help you out! 😄

**Reply by DebbieU** [reply]
I just realized I had a commit waiting to happen on my GitHub desktop. Once I committed that, everything worked! Now to close this thread!!

**Reply by radar-blip** [reply]
Thats great! Good work!! 
Not sure you can close this thread, it will remain for others who are having the same problem.
